app/src/main/java/com/example/concordia_campus_guide/BuildingCode.java

```diff
@@ -0,0 +1,68 @@
+package com.example.concordia_campus_guide;
+
+public enum BuildingCode {
+    AD,
+    BB,
+    BH,
+    CC,
+    CJ,
+    DO,
+    FC,
+    GE,
+    HA,
+    HB,
+    HC,
+    HU,
+    JR,
+    PC,
+    PS,
+    PT,
+    PY,
+    QA,
+    RA,
+    RF,
+    SC,
+    SH,
+    SI,
+    SP,
+    TA,
+    VE,
+    VL,
+    B,
+    CB,
+    CI,
+    CL,
+    D,
+    EN,
+    ER,
+    EV,
+    FA,
+    FB,
+    FG,
+    GA,
+    GM,
+    GN,
+    GS,
+    H,
+    K,
+    LB,
+    LD,
+    LS,
+    M,
+    MB,
+    MI,
+    MU,
+    P,
+    PR,
+    Q,
+    R,
+    RR,
+    S,
+    SB,
+    T,
+    TD,
+    V,
+    VA,
+    X,
+    Z
+}
```

Comment:
https://android.jlelse.eu/android-performance-avoid-using-enum-on-android-326be0794dc3?gi=512c816544d0 please see this, enum is not very good as they consume a lot of resources, Android recommend using other types of structure such as TypeDef.

Reply:
this modification would cost us a lot of time... I will keep it in mind and change it next sprint